src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
its not reset

Reply:
@danylo-onishchuk `Normalize` describes this better? Next time will use it.